nodejs/BUILD.bazel

```diff
@@ -27,3 +28,11 @@ toolchain_type(
     name = "toolchain_type",
     visibility = ["//visibility:public"],
 )
+
+[
+    platform(
+        name = key,
+        constraint_values = values.compatible_with,
+    )
+    for key, values in PLATFORMS.items()
+]
```

Comment:
Suggestion/question: wouldn’t it be more self-explanatory if we were to make a `platforms` directory and move this inside the BUILD file?

Reply:
eh, it's a pretty trivial difference. I'd say that the data is already encoded in the graph by `bazel query 'kind(platform, @rules_nodejs//nodejs:all)'` and it's not enough semantics to be worth another BUILD file. It makes all the target names longer and means our distribution contents needs another filegroup. Users barely ever type this (and rules_docker is gaining transitions so you won't need to use `--platforms` there anymore.)